src/peers.py

```
@@ -0,0 +1,534 @@
+def _now_utc() -> str:
+    return datetime.datetime.now(datetime.timezone.utc).strftime("%Y-%m-%d %H:%M:%S")
+
+
+async def _notify(env, user_id: str, type_: str, title: str, message: str,
+                  related_id: Optional[str] = None) -> None:
+    try:
+        await _worker()._create_notification(
+            env, user_id, type_, title, message,
+            related_id=related_id, category="system",
+        )
+    except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)